documentation/docs/getting_started/security_101.md

```diff
@@ -11,40 +11,44 @@ keywords:
 ---
 # Security 101
 
-This section provides a checklist of steps for running a reliable and secure node.
+This section explains how and why node security is important while also providing a checklist of steps for running a reliable and secure node.
 
 :::info
-Please note that servers that are reachable from the Internet are a constant target from security challengers.  Please, make sure you follow the minimum security essentials summarized in this article.
+Servers that are reachable from the Internet are a constant target from security challengers. Please, make sure you follow the minimum security essentials found here.
```

Comment:
"found here" doesn't seem to add much to the sentence.  Perhaps it would be better left out